### ISSUE TITLE
wont read give me answers

### ISSUE THREAD

**Issue by bnicer** [sushant10/HQ_Bot#101]
![image](https://user-images.githubusercontent.com/6856878/39448040-4cdf35f2-4c89-11e8-840a-afae0bd95ac4.png)

**Comment by U-r-Unknown** [comment]
I have working bot which works on all trivia games. No worries about ban as this is PC software. No need of any programming knowledge. If interested mail me onlineuse321@gmail.com

**Comment by bnicer** [comment]
I was able to fix it, I'm not sure how since I did so many install like visual studio and I install anaconda on PC. But the last 2 step I did was download pytesseract and tesseract for Windows. I installed pytesseract by pip install pytesseract then pip install tesseract and it was perfectly working after.

**Comment by wraith1821** [comment]
Any idea as to what else you installed?

**Comment by bnicer** [comment]
I did install
1.) Tesseract for Windows
https://github.com/tesseract-ocr/tesseract/wiki
2.) pip install pytesseract
3.)pip install tesseract
That's the last steps I made

**Comment by wraith1821** [comment]
Yeah, did all that.... Thanks anyways.  Looks like a few other people have the same issue.  Ill wait for an answer there.  I've installed a slew of junk to no avail. :/
